perfeed/tools/pr_summarizer.py

```diff
@@ -57,14 +59,45 @@ async def run(
         )
         return pr_summary, pr_metadata
 
+    
+    def _comments_to_thread(self, pr_comments: list[PRComment]) -> str:
```

Comment:
I feel like this function (even though it's internal) deserves a unit test since the output (relationship of threads) matters a lot to LLM reasoning